src/components/Toolbar/index.js

```diff
@@ -23,6 +23,28 @@ export default class Toolbar extends Component {
     this.props.onTogglePencil();
   };
 
+  handlePencilColorPickerClick = (e) => {
+    e.stopPropagation();
+    let hexColor = getComputedStyle(document.documentElement)
+      .getPropertyValue('--pencil-color')
+      .trim()
+      .substring(1);
+    if (hexColor.length === 3) {
+      hexColor = hexColor
+        .split('')
+        .map(function (hex) {
+          return hex + hex;
+        })
+        .join('');
+    }
+    this.pencilColorPicker.value = '#' + hexColor;
+    this.pencilColorPicker.click();
+  };
+
+  handlePencilColorPickerChange = (e) => {
+    document.documentElement.style.setProperty('--pencil-color', e.target.value);
```

Comment:
i'm fine w/ storing this global state on dom instead of in react, could one day persist to browser storage too